### PR TITLE
📝 Document how to release

### PR DESCRIPTION
**Pull request by ruksi**
document this here as I surely will forget this for the next time

let me know if it's good enough or you want to mention using a separate PR to do a release :pray: 

also includes a minor change to exclude individual (merge) commits from the changelog :pinching_hand:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,5 +83,30 @@ mypy --strict --install-types --show-error-codes minique
 REDIS_URL=redis://localhost:6379/0 pytest .
 ```
 
+### Release
+
+```shell
+# decide on a new versio number and set it
+vim minique/__init__.py
+__version__ = "0.9.0"
```
Comment (akx) [inline README.md:91]: Also: `hatch version minor` (and friends) :)

File: README.md [hunk 2]
```diff
@@ -83,5 +83,30 @@ mypy --strict --install-types --show-error-codes minique
...
+# or
+npm install auto-changelog
+node node_modules/auto-changelog/src --commit-limit=0
+
+# undo changes changelog generation did to the older entries
+
+git add CHANGELOG.md
+git commit --amend --no-edit
+git tag -d v0.9.0
+git tag -m "v0.9.0" -a v0.9.0
```
[the remark this answers is not in the document]
Comment (akx) [reply README.md:107]: Might need some additional flags, not sure. ... or another tool at some point.

File: README.md (hunk 1 above)
Comment (ruksi) [reply README.md:91]: `Command 'hatch' not found` :disappointed:

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ruksi) [reply README.md:107]: seems to work alright now, I'll splat the edits and lets see how it works for the next release

**Comment by ruksi** [comment]
revised!

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ruksi) [reply README.md]: I like the GA-based [Release Drafter](https://github.com/release-drafter/release-drafter) as it creates GitHub releases that look nicer than a Markdown file, but I haven't tried many alternatives :shrug: 

usage example: https://github.com/ruksi/genvi/blob/main/.github/workflows/release-drafter.yml

[the remark this answers is not in the document]
Comment (ruksi) [reply README.md]: you need to remember to actually _release_ the draft it continually updates though

**Review by akx — APPROVED** [review]

**Pull request merged**